sqlglot/dialects/postgres.py

```diff
@@ -304,7 +304,7 @@ def _parse_date_part(self) -> exp.Expression:
             return self.expression(exp.Extract, this=part, expression=value)
 
     class Generator(generator.Generator):
-        INTERVAL_ALLOWS_PLURAL_FORM = False
```

Comment:
I tested against both engines (postgres, redshift), this was invalid as plural forms are allowed.